src/MomentoTaggedCache.php

```diff
@@ -19,7 +18,8 @@ public function put($key, $value, $ttl = null): bool
         $newKey = self::createNewKey($tags, $key);
         $hashedKey = hash("sha256", $newKey);
         foreach ($tags as $tag) {
-            $hashedKeyResponse = $this->store->setAdd($cacheName, $tag, $hashedKey, true, self::maxTtl);
+            # Divide PHP_INT_MAX by 1000 so that when mxTtl is converted to milliseconds by ttlToMillis, the return value is still int not float.
```

Comment:
this comment seems like it goes with the variable assignment up above, rather than here?